fuse-products/src/main/java/software/tnb/product/cq/application/QuarkusApp.java

```diff
@@ -112,7 +145,11 @@ private void customizeProject(List<Dependency> dependencies) {
         Model model = Maven.loadPom(pom);
 
         // Append the camel platform bom (quarkus bom already present)
-        Dependency camelQuarkusBom = new Dependency();
+        // Check if the cq bom is already present in the dependencies management, if it is, then it is overriden
+        Dependency camelQuarkusBom = model.getDependencyManagement().getDependencies().stream()
+            .filter(d -> d.getArtifactId().equals(QuarkusConfiguration.camelQuarkusPlatformArtifactId())).findFirst().orElse(new Dependency());
+        model.getDependencyManagement().getDependencies().remove(camelQuarkusBom);
```

Comment:
what about `<...>.findFirst().ifPresent(model.getDependencyManagement().getDependencies()::remove)`? it might look less suspicious :D